modules/likes.php

```diff
@@ -896,6 +896,9 @@ function is_likes_visible() {
 			if ( post_password_required() )
 				$enabled = false;
 
+      if ( in_array( 'get_the_excerpt', (array) $wp_current_filter ) )
```

Comment:
This should be indented with tabs, and also use braces for the conditional please